### DISCUSSION TITLE
0.3.0

### DISCUSSION THREAD

**Discussion by mcndt** [mcndt/obsidian-toggl-integration#31] [Releases]
### ✨ Features

- Sidebar view now shows tags on active time entry.
- You can now add tags to a new time entry when entering the description.
- Adding tags to a time entry uses autocompletion using the Toggl workspace's existing tags.

### ⚙️ Internal

- Plugin now immediately refreshes current timer status when starting/stopping a time entry and on plugin load.

<hr /><em>This discussion was created from the release <a href='https://github.com/mcndt/obsidian-toggl-integration/releases/tag/0.3.0'>0.3.0</a>.</em>

**Comment by sirlaughalot90** [comment]
Thank you so much for implementing the tag features! Very useful and perfectly implemented.

**Reply by mcndt** [reply]
Let me know if you run into any bugs or flaws with tags!

**Reply by sirlaughalot90** [reply]
I sure will.
